report/renderer/performance-category-renderer.js

```diff
@@ -332,9 +332,9 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
       // Toggle class/hidden state based on filter choice.
       radioEl.addEventListener('input', _ => {
         for (const elem of categoryEl.querySelectorAll('label.lh-metricfilter__label')) {
-          elem.classList.toggle('lh-metricfilter__label--active', elem.htmlFor === elemId);
+          elem.classList.toggle('lh-metricfilter__label--active', elem.dataset.acronym === acronym);
```

Comment:
```suggestion
          elem.classList.toggle(
            'lh-metricfilter__label--active', elem.dataset.acronym === acronym);
```

slap a newline here if the reason for `acronym` variable was eslint maxlen

Reply:
i know it'd be a very PAUL thing to change code in order to escape eslint maxlen, but i swear that's not why i did that.  😎 
but yeah i'm totes fine with keeping things as `metric.acronym` so back we go. :)